pkg/storaged/client.js

```diff
@@ -171,6 +171,7 @@ function init_proxies () {
     client.blocks_crypto = proxies("Encrypted");
     client.blocks_swap = proxies("Swapspace");
     client.iscsi_sessions = proxies("ISCSI.Session");
+    client.vdo_vol = proxies("VDOVolume");
```

Comment:
The convention is to use plural here: `vdo_vols`.  Or `lvols_vdo` if this is an extension interface to LogicalVolume (and this should be moved next to `client.lvols = proxies("LogocalVolume")`.

Reply:
Renamed to vdo_vols.